components/Modal/PendingChanges.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class PendingChanges extends React.Component {
+
+  // dummy data
+  state = {
+    componentUpdates: [
+      { componentOld: null, componentNew: 'rsync-3.0-17.317' },
+      { componentOld: 'httpd-2.3-45.el7', componentNew: 'httpd-2.4-45.el7' },
+      { componentOld: 'basesystem-10.0-7.el7', componentNew: null },
+    ],
+  };
+
+  componentDidMount() {
+    $(this.modal).modal('show');
+    $(this.modal).on('hidden.bs.modal', this.props.handleHideModal);
+  }
+
+  handleSaveChanges() {
+    $('#cmpsr-modal-pending-changes').modal('hide');
+  }
+
+  render() {
+    return (
+      <div
+        className="modal fade"
+        id="cmpsr-modal-pending-changes"
+        ref={(c) => { this.modal = c; }}
+        tabIndex="-1"
+        role="dialog"
+        aria-labelledby="myModalLabel"
+        aria-hidden="true"
+      >
+        <div className="modal-dialog">
+          <div className="modal-content">
+            <div className="modal-header">
+              <button
+                type="button"
+                className="close"
+                data-dismiss="modal"
+              >
+                <span className="pficon pficon-close"></span>
+              </button>
+              <h4 className="modal-title" id="myModalLabel">Changes Pending Save</h4>
+            </div>
+            <div className="modal-body">
+              <form className="form-horizontal">
+                <div className="form-group">
+                  <label
+                    className="col-sm-3 control-label"
+                  >Recipe</label>
+                  <div className="col-sm-9">
+                    <p className="form-control-static">{this.props.recipe}</p>
+                  </div>
+                </div>
+                <div className="form-group">
+                  <label
+                    className="col-sm-3 control-label"
+                    htmlFor="textInput-modal-markup"
+                  >Comment</label>
+                  <div className="col-sm-9">
+                    <textarea
+                      id="textInput-modal-markup"
+                      className="form-control"
+                      rows="1"
+                      value={this.props.recipe.comment}
+                    />
+                  </div>
+                </div>
+                <div className="alert alert-info">
+                  <span className="pficon pficon-info"></span>
+                  Only changes to selected components are shown. <a href="#" className="alert-link">View all changes.</a>
+                </div>
+                <strong>Updates</strong>
```

Comment:
Please change `<strong>Updates</strong>` to `<strong>Pending Changes</strong><span class="text-muted"> (most recent first)</span>`

This captures the suggestion that Steve made on the design, and also avoids using "update" to mean too many different things.